addon/components/object-list-view.js

```diff
@@ -1032,6 +1050,62 @@ export default FlexberryBaseComponent.extend(
       this.get(action)(column, this.get('componentName'));
     },
 
+    /**
+      Clear selected rows.
+
+      @method actions.clearSelectedRecordsMobile
+      @public
+    */
+    clearSelectedRecordsMobile() {
+      let componentName = this.get('componentName');
+      let contentWithKeys = this.get('contentWithKeys');
+      let selectedRecords = this.get('selectedRecords');
+      let selectedRows = contentWithKeys.filterBy('selected', true);
+      for (let i = 0; i < selectedRows.length; i++) {
+        let recordWithKey = selectedRows[i];
+        let selectedRow = this._getRowByKey(recordWithKey.key);
+
+        if (selectedRow.hasClass('active')) {
+          selectedRow.removeClass('active');
+        }
+
+        selectedRecords.removeObject(recordWithKey.data);
+        recordWithKey.set('selected', false);
+
+        this.get('objectlistviewEventsService').rowSelectedTrigger(componentName, recordWithKey.data, selectedRecords.length, false, recordWithKey);
+      }
+    },
+
+    /**
+      Delete selected rows.
+
+      @method actions.deleteSelectedRowMobile
+      @public
+    */
+    deleteSelectedRowMobile() {
```

Comment:
Instead of adding the `...Mobile` suffix everywhere, move the changes to `components/mobile/object-list-view.js`.